Documentation/concepts/networking/ipam/eni.rst

```diff
@@ -387,6 +387,9 @@ the cilium-operator. This allows the user to supply a custom limit.
 Additionally the limits can be updated via the EC2 API
 by passing the ``--update-ec2-apdater-limit-via-api`` CLI flag. This will require an additional EC2 IAM permission:
 
+Additionally the limits can be updated via the EC2 API v2
+by passing the ``--update-ec2-adapter-limit-via-api`` CLI flag. This will require an additional EC2 IAM permission:
+
```

Comment:
This is duplicating the paragraph above. I think we should change the parameter name in the paragraph instead.

Reply:
> This is duplicating the paragraph above. I think we should change the parameter name in the "old" paragraph instead.

I am not sure what you meant by 'parameter name' change.
In the new paragraph I mentioned it as 'EC2 API v2' and also the command line flag 'update-ec2-adapter-limit-via-api' is different from 'update-ec2-apdater-limit-via-api'.

What else do you think we should be able to change here. Since both denotes the same functionality other than the typo, I am not sure what else make sense here.

Reply:
I have made some tweaks, let me know if that would work and make sense.